### ISSUE TITLE
get_accounts shows zero balance

### ISSUE THREAD

**Issue by gropox** [GolosChain/golos#153]
get_accounts returns in JSON zero balance 

```
    "balance": "0.000 GOLOS",
    "savings_balance": "0.000 GOLOS",
    "sbd_balance": "0.000 GBG",
    "sbd_seconds": "0",
```

get_account_balances returns right balance

![image](https://user-images.githubusercontent.com/26039472/28498673-e3a897ea-6fa2-11e7-8edc-e36412ff0f4d.png)

**Comment by Nemo1369** [comment]
Not a bug. Account balances are stored in the separate account_balance_object now. It can be with https://github.com/GolosChain/golos/blob/golos-v0.17.0/libraries/application/database_api.cpp#L75

**Comment by gropox** [comment]
This breaks compatibility with existing scripts.  Better would fill these tags in structure.  Then no needs to rewrite 3rd party applications.

**Comment by Nemo1369** [comment]
@gropox That means we would need to list all the balances as the account_object class data members. So the account_object will become the a lot bigger and hard to retrieve. Someone would suggest to create lightweight_account_api_object, just like an extended_account class exists. That would be a code smell.

The whole API is basically going to be redesigned, so this is the beginning.

Of course, all the changes are going to be highlighted in documentation/API reference.

**Comment by gropox** [comment]
No, it would be enough if only GBG and GOLOS are shown. UIA is something new. The applications using UIA are not written yet. It is just about existing applications. 

I will correct my scripts. But others should be at least informed.

**Comment by Nemo1369** [comment]
@gropox Alright, keeping the backward compatibility for one more hardfork is a fine idea.

**Comment by Nemo1369** [comment]
Closed with eed153b

Backward compatibility implemented.
